tier1_standards/ci.md

```diff
@@ -22,6 +22,8 @@ The existence of unit tests is highly encouraged as part of this standard, but n
 See a guide on how to set up multiple CI options from the [STScI Training Library](https://innerspace.stsci.edu/display/INSTL/Continuous+Integration+and+Testing)
 
 ## Useful Links
-- [Travis Tutorial](https://docs.travis-ci.com/user/tutorial/)
```

Comment:
I'm low-key impressed that the ST Style Guide has already been updated to remove Travis as an option.  Good job team.